mgmtsystem_probability/model/mgmtsystem_probability.py

```diff
@@ -0,0 +1,47 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2015 - Present
+#    Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp.tools.translate import _
+from osv import fields, orm
+
+_CATEGORIES = [
+    ("hazard", _("hazard")),
+    ("security", _("security")),
+]
```

Comment:
> _("...") wont work the way you expect it to here.

I expected the parser that export translation to catch it. Openerp does the translation later. There is no need to call the translation method with the context at runtime because it will try to translate the translated string later probably.

But I kind of agree with the lambda part if you want to inherit it and update the list yet, the list could be static in the module anyway. Which mean that you don't have to recreate the list and translate the list multiple times.